packages/cli/src/cmds/validator/options.ts

```diff
@@ -248,7 +248,7 @@ export const validatorOptions: CliCommandOptions<IValidatorCliArgs> = {
   },
 
   "builder.boostFactor": {
-    type: "number",
+    type: "string",
```

Comment:
It does not look like yargs supports bigint based on https://github.com/yargs/yargs/issues/440 and looking at type options.